packages/conf-libsamplerate/conf-libsamplerate.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+homepage: "http://www.mega-nerd.com/SRC/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: "libsamplerate dev team"
+license: "BSD"
+build: ["pkg-config" "--exists" "samplerate"]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  ["libsamplerate-dev"] {os-family = "debian"}
+  ["libsamplerate-dev"] {os-family = "ubuntu"}
```

Comment:
Is this really needed?

Reply:
If we wish to support Ubuntu-derived distribution, yes.

Opam gets the value of `os-family` from the first value of `ID_LIKE` in `/etc/os-release`, which for Ubuntu-derivered distributions might be `ubuntu` first.

Reply:
Ha, tricky.